pull/github.go

```diff
@@ -137,17 +135,17 @@ type GitHubContext struct {
 	collaborators []*Collaborator
 	permissions   map[string]Permission
 	teams         map[string]Permission
-	teamIDs       map[string]int64
 	membership    map[string]bool
 	statuses      map[string]string
 	labels        []string
+	pushedAt      map[string]time.Time
```

Comment:
This could actually be cached globally: once we know the push time for a specific commit in a specific repository, it shouldn't change. I think for now it's easiest to re-compute it on each request, but if the extra requests become a problem, this might be a good optimization.